samples/dockers/Dockerfile.dev

```diff
@@ -13,8 +13,8 @@ USER autogen-dev
 RUN cd /home/autogen-dev && git clone https://github.com/microsoft/autogen.git
 WORKDIR /home/autogen-dev/autogen
 
-# Install autogen (Note: extra components can be installed if needed)
-RUN sudo pip install -e .[test]
+# Install autogen in editable mode (Note: extra components can be installed if needed)
+RUN sudo pip install -e .[test,teachable,lmm,graphs]
```

Comment:
I'm not sure about "graphs". I did a search in the repo and it is not used anywhere.